frontend/src/util/math.ts

```diff
@@ -0,0 +1,33 @@
+import { round } from "lodash";
+
+/** format map data value */
+export function formatValue(
+  value = 0,
+  min = 0,
+  max = 1,
+  compact = true,
+): string {
+  if (min >= 0 && max <= 1) return round(value * 100, compact ? 1 : 3) + "%";
```

Comment:
Found myself doing this same thing many times, so extracted it to a util func.

Reply:
Adding comments. Yes, basically if all the values are between 0 and 1, the value clearly represents a percentage (I'm hoping this assumption is correct for all measures, it seems to be), and so we should format it as such.